decidim-core/lib/gem_overrides/webpacker/runner.rb

```diff
@@ -3,7 +3,7 @@
 # This file overrides the webpacker/runner require in order to customize the
 # runner when it is started.
 
-require "#{Gem.loaded_specs["webpacker"].full_gem_path}/lib/webpacker/runner"
+require "#{Gem.loaded_specs["shakapacker"].full_gem_path}/lib/webpacker/runner"
```

Comment:
Couldn't we remove this whole override when we have this after the 6.6.x release?
https://github.com/decidim/decidim/blob/82617f28e83c7ebe6b26932321509d6b99f5533c/decidim-core/lib/decidim/core/engine.rb#L59

Reply:
OK, maybe I don't understand the purpose of that configuration then.

Isn't that configuration exactly what is supposed to make the runner overrides unnecessary?

Reply:
The problem there is not the configuration file itself, but i am thinking about which is in the same file, i and i need to fully understand what is exactly doing ...

https://github.com/decidim/decidim/blob/82617f28e83c7ebe6b26932321509d6b99f5533c/decidim-core/lib/decidim/webpacker/runner.rb#L22

Reply:
Oh OK, got it. Makes more sense then.

That is generating the Tailwind configuration on the fly based on the template. Tailwind is quite a beast by itself because it compiles the CSS based on the Tailwind classes used in `.js` and `.erb` files (and also some `.rb` files such as helpers, cells, form builders, etc.). Through this Tailwind configuration, you can tell the Tailwind form which files it should read the classes and also which files it should watch during development.

Reply:
Yet, here we are .. Let;s see what we can do, so we can either remove or replace the current status quo.

Reply:
I think we can handle this issue in another PR.